flutter/lib/src/jvm/jvm_exception.dart

```diff
@@ -256,8 +256,10 @@ class JvmException {
       frames.add(trimmed);
     }
 
-    final thisExceptionFrames =
-        thisException.map((e) => JvmFrame.parse(e)).toList(growable: false);
+    final thisExceptionFrames = thisException
+        .where((line) => line.trim().isNotEmpty)
```

Comment:
Maybe explain with a comment why it's being done like that?
```suggestion
        // Sometimes stringified exceptions from the native side have
        // empty lines. Discard those!
        .where((line) => line.trim().isNotEmpty)
```